libs/service-portal/graphql/src/hooks/alertBanners/useAlertBanners.ts

```diff
@@ -0,0 +1,63 @@
+import { useEffect, useRef, useState } from 'react'
+import Cookies from 'js-cookie'
+import { useQuery } from '@apollo/client'
+import { AlertBanner as AlertBannerSchema } from '@island.is/api/schema'
+import { GET_SERVICE_PORTAL_ALERT_BANNERS_QUERY } from '../../lib/queries/alertBanners'
+import { useLocale } from '@island.is/localization'
+
+export type AlertBannerType = AlertBannerSchema & { bannerId: string }
+
+// Taken from here: https://stackoverflow.com/a/7616484
+const stringHash = (str: string): number => {
```

Comment:
This same method is defined and used in web. Can we move this to a shared utils library?

Reply:
Yeah, totally. I'll do that in a follow up PR (I also need to start using sortAlpha shared utils library instead of using the copy pasted version on the web)